interview-questions.md

```diff
@@ -8,36 +8,91 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: (I have unfortunately done this in one of the Rails challenges 😅) In order to add a foreign key to an existing table, you would need to first create a migration
+    'rails g migration migration_name'
+Now with a new migration file, we can name what we're doing (adding a column) and add that to the file 
+    'add_column :students, :cohort_id'
+After running 'rails db:migrate' the Student model will have the foreign key
 
-Researched answer:
+Researched answer: If you've just created your model and forgot to make a foreign key, an easy workaround is to run
+'rails db:rollback'
```

Comment:
While rollback has a time a place to be used properly, I would recommend generating a migration file and fixing it that way. Rollback modifies your migration history and can lead to possible problems if not careful, whereas creating a new migration adds to the story of how your database has come to be what it is.